projects/clr/rocclr/device/rocm/rocdevice.cpp

```diff
@@ -2588,7 +2600,10 @@ bool Device::GetSvmAttributes(void** data, size_t* data_sizes, int* attributes,
           break;
       }
     }
-
+    // Only call svm_attributes_get if there are SVM attributes to query
+    if (attr.empty()) {
```

Comment:
Why do we need this check at all? There is already a check at the top level for 0 attributes.